### PR TITLE
Add stack

### PR DESCRIPTION
**Pull request by iamneha**
#4 @aktech review it

### CODE REVIEW

File: pydsa/tests/test_stack.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+def test_stack():
+    q = stack()
+    q.push(5)
+    q.push(9)
+    q.push(12)
+    q.push(20)
+    assert q.pop() == 20
+    assert q.pop() == 12
+    assert q.size() == 2
```
Comment (aktech) [inline pydsa/tests/test_stack.py]: Add test for is_empty also

File: pydsa/stack.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    4
+    >>> q.pop()
+    1
+    >>> q.pop()
+    19
+    """
+    def __init__(self):
+        self.List = []
+
+    def isEmpty(self):
```
Comment (aktech) [inline pydsa/stack.py]: isEmpty --> is_empty

File: pydsa/stack.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+class stack(object):
```
Comment (aktech) [inline pydsa/stack.py]: Class name should start with capital letter

File: pydsa/stack.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
+class Stack(object):
+    """
+    >>> from pydsa import stack
+    >>> q = Stack()
```
Comment (aktech) [inline pydsa/stack.py]: `s` would be a better variable than `q`

File: pydsa/tests/test_stack.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
+from pydsa.stack import Stack
+
+
+def test_stack():
+    q = Stack()
```
Comment (aktech) [inline pydsa/tests/test_stack.py]: `s` would be a better variable than `q`

**Comment by aktech** [comment]
Please fix merge conflicts.

**Comment by iamneha** [comment]
@aktech merged conflict and did all changes

File: pydsa/stack.py [hunk 6]
```diff
@@ -0,0 +1,38 @@
+class Stack(object):
+    """
```
Comment (aktech) [inline pydsa/stack.py]: Please add docstring here.

**Comment by aktech** [comment]
It's good to go except this [comment](https://github.com/pydsa/pydsa/pull/61/files#r59789857).

**Comment by iamneha** [comment]
@aktech just a minute

**Comment by aktech** [comment]
Looks good. +1

**Pull request merged**